src/Component/About/About.tsx

```diff
@@ -5,22 +5,22 @@ import { BasicBanner } from "../BasicBanner/BasicBanner";
 export const About = () => {
   return (
     <BasicBanner>
-        <div className="about_container">
+      <div className="about_container">
         <section className="about_logo">
-            <img src={ABOUT.LOGO} />
+          <img src={ABOUT.LOGO} />
         </section>
 
         <main className="about_details">
-        <section className="mobile">
+          <section className="mobile">
             <img src={ABOUT.APP_STORE} />
             <img src={ABOUT.GOOGLE_PLAY} />
-        </section>
-        <section>
+          </section>
+          <section className="about_title">
             <h1>{ABOUT.TITLE}</h1>
-            <h3>{ABOUT.DESC}</h3>
-            </section>
+            <h3 dangerouslySetInnerHTML={{ __html: ABOUT.DESC }} />
```

Comment:
why did we use 'dangerouslySetInnerHTML' here?

Reply:
I did it to create new row using <br> inside the text that comes from {ABOUT.DESC}. I did not find another way to do it from the text itself.

Reply:
its best to avoid using 'dangerouslySetInnerHTML' for security risks, I know we havent covered it in our class but just for future reference (u can use different libraries that can render html tags with texts).